deps/wazuh_testing/wazuh_testing/modules/fim/event_monitor.py

```diff
@@ -393,23 +441,50 @@ def check_fim_event(file_monitor=None, callback='', error_message=None, update_p
                        callback=generate_monitoring_callback(callback), error_message=error_message)
 
 
-def detect_initial_scan(file_monitor):
+def get_messages(callback, timeout=T_10, update_position=True, max_events=MAX_EVENTS_VALUE, error_message=None):
+    """Look for as many events of a given type as possible.
+
+    Args:
+        callback (object): callback that returns the events that are being looked for.
+        timeout (int): Timeout that will be used to get the dbsync_no_data message.
+        update_position (bool, optional): True if we pop items from the queue once they are read. False otherwise.
+                Default `True`
+        max_events (int, optional):
+
+    Returns:
+        A list with all the events.    """
+
+    wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+    error_message = error_message if error_message is not None else 'Did not receive expected events'

```

Comment:
This format is invalid. It requires an else clause to use conditional assignment

Reply:
It is enough to validate that it is "NONE". 
That's what I was pointing to with my comment. I understand that it doesn't apply, but I thought it was clear what I was pointing to since we both know that this can't be done by syntax.

Reply:
Talked about this in Sync